### DISCUSSION TITLE
Is every game guaranteed winnable?

### DISCUSSION THREAD

**Discussion by melissaboiko** [Pheonyxior/Pirate-Solitaire-Git-Repo#18] [Q&A]
Hi! I'm having trouble with a seed, and I don't want to give up if there's a solution I'm not seeing.  But I don't want to spend more time with it if some games are unwinnable.  Are there checks or tests that ensure at least one solution exists?

(thanks for the game, I play it a lot!)

**Comment by Pheonyxior** [comment]
Hello, thanks for your kind comment. ^^ 

The truth is, I don't know either ! I asked myself the same question while developing the game, and it feels like a big undertaking of its own. I remember looking up discussions on Reddit about the original Microsoft free cell solitaire in the same vein. 

I do think you could manufacture an impossible game, like putting every 2 and 1 at the bottom of the stacks, covered by 3 beasts and 3 pirates, and then putting the other 3 pirates at the top of the stacks. 

I imagine that's not what your game looks like though, I'd be very curious giving it a go myself

**Reply by Pheonyxior** [reply]
Also, you can copy a seed on a note on your phone or something and then paste it back in the seed edit panel, in case you'd like to come back to a tough deal later.

**Reply by melissaboiko** [reply]
Thanks! Yeah I kept coming back to it and eventually found a solution, it was `_6953654777886327769` . I'm yet to find a game that couldn't be solved eventually (though not always I'm super persistent so who knows).
